docs/logging.asciidoc

```diff
@@ -0,0 +1,135 @@
+[[logging-integrations]]
+=== Logging integrations
+
+The Elastic APM agent provides integrations with both the default Python logging library,
+as well as http://www.structlog.org/en/stable/[`structlog`].
+
+[[logging]]
+==== `logging`
+
+For Python 3.2+, we use https://docs.python.org/3/library/logging.html#logging.setLogRecordFactory[`logging.setLogRecordFactory()`]
+to decorate the default LogRecordFactory to automatically add new attributes to
+each LogRecord object:
+
+* `elasticapm_transaction_id`
+* `elasticapm_trace_id`
+* `elasticapm_span_id`
+
+This factory also adds these fields to a dictionary attribute,
+`elasticapm_labels`, using the official ECS https://www.elastic.co/guide/en/ecs/current/ecs-tracing.html[tracing fields].
+
+You can disable this automatic behavior by setting
```

Comment:
How about

    You can disable this automatic behavior by using the
    <<config-generic-disable-log-record-factory,`disable_log_record_factory`>> setting
    in your configuration

The reason for this being that how exactly the configuration looks depends on the usage (environment variable, Django/Flask config, inline config).

Reply:
Done.